app/src/main/res/values/strings.xml

```diff
@@ -127,19 +127,14 @@
     </string>
     <string name="about_background_heading">Background</string>
     <string name="about_background_body_text">
-        We\'ve all been there: It's one of those group situations, where, after a little while, the
-        group branches off into pairs. Jenny, the runner, goes
-        with Fred, also a runner. James always watches indie flicks. He\'ll end up with April
-        because she does the same.\n
+        We've all been there: messages that go nowhere; likes, winks, and pokes sent with no response; 
+        conversations that fizzle and die. How can it be so hard to meet someone you like and who actually likes you back?\n
         \n
-        When we get to a big  city with a big dating pool, we suddenly see the creme of the crop.
-        Some of us won\'t settle for anything less than chasing unattainable "gods" and
-        "goddesses." Beeline slots you with only a small number of matches that you actually have
-        a chance with.\n
+        Fortunately, Beeline's got your back. It slots you with only a small number of matches that are 
+        genuinely interested in you.\n
         \n
-        Avoid wasting time by sending hundreds of messages with zero responses.
-        Avoid getting your inbox flooded by thousands of messages. Get focused right on a hand-
-        selected number of people that are your high value matches.
+        Avoid wasting time by sending hundreds of messages with zero responses. Get focused on a chosen few 
```

Comment:
That's clear and short with a fun tone.

Nitpicks:
- "a chosen few" -> "your chosen few" No algorithm in the usual marketing sense made the decision. Both users decided on whom they like. It's good to restate the essence of "you are the algorithm."
- "create your own connections" seems a bit vague. Something that expresses a serious beginning of the dating process seems better.